tests/manual-tests/framework/preferences/assortedContent/index-jQueryUI.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
```

Comment:
The idea with examples is that they are illustrative of how to do something in particular. In this case it would be switching to the jQuery widgets. And the broader hope is to move or some link these to concrete documentation/tutorials in our docs site.